include/nanorange/algorithm/rotate_copy.hpp

```diff
@@ -9,11 +9,12 @@
 
 #include <nanorange/ranges.hpp>
 #include <nanorange/algorithm/copy.hpp>
+#include <nanorange/detail/algorithm/return_types.hpp>
```

Comment:
These should be alphabetical (my fault, sorry)

Reply:
Done.